### PR TITLE
docs: Minor language and clarity updates

### PR DESCRIPTION
**Pull request by itsthejoker**
Minor cleanup for the readme and fixing a typo in `generate-renovate-config.yaml`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,17 @@ This is [Shareable Config Presets](https://docs.renovatebot.com/config-presets/)
...
+---
+
 Otherwise:
 
-0. If the repo is too much outdated - prefer to update manually as much as you can before merging Renovate init PR.
-1. Activate [Renovatebot Github App](https://github.com/marketplace/renovate) for your repo or ask Github org administrators.
-2. Renovate will create [init [PR](https://docs.renovatebot.com/getting-started/installing-onboarding/#repository-onboarding) for the new repo - open it and check that it has:
+0. Manually update as much as you can before moving forward. You will have the best experience with Renovate if you start with a fully-updated repo.
+1. Activate [Renovatebot Github App](https://github.com/marketplace/renovate) for your repo or ask your GitHub org administrators to enable it.
+2. Renovate will create an [init [PR](https://docs.renovatebot.com/getting-started/installing-onboarding/#repository-onboarding) for the new repo - open it and check that it has:
```
Comment (MaxymVlasov) [inline README.md:39]:
```suggestion
2. Renovate will create an [init PR](https://docs.renovatebot.com/getting-started/installing-onboarding/#repository-onboarding) for the new repo - open it and check that it has:
```

File: README.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ This is [Shareable Config Presets](https://docs.renovatebot.com/config-presets/)
 ## Usage
 
 <!-- markdownlint-disable-next-line no-inline-html -->
-<details><summary><u>If Renovate has already been activated for repo</u></summary>
+<details><summary>If Renovate has already been activated for repo:</summary>
```
Comment (MaxymVlasov) [inline README.md:15]: A colon has a tendency to mislead users

```suggestion
<details><summary><u>If Renovate has already been activated for repo</u></summary>
```

Comment (itsthejoker) [reply README.md]: The underline doesn't render on github, which is why I removed it. I'm indifferent towards the colon though 😄

Comment (MaxymVlasov) [reply README.md]: Ah, Okay, interesting, why then in MD ext in VS Code with Github style setting it renders 🤔 
Nice catch!

**Review by MaxymVlasov — APPROVED** [review]
>Error: A branch or tag with the name 'update-docs' could not be found

The weird error didn't see before from forks in other OSS repos. I'll leave it as is for now

**Pull request merged**